include/swift/SIL/SILDeclRef.h

```diff
@@ -384,7 +384,11 @@ struct SILDeclRef {
   /// True if the function should be treated as transparent.
   bool isTransparent() const;
   /// True if the function should have its body serialized.
-  IsSerialized_t isSerialized() const;
+  bool isSerialized() const;
```

Comment:
In a follow-up PR we should also introduce a predicate: `bool isAnySerializedKind()`.

The places where we now have to write double negation `!isNotSerialized()` are really hard to read. At least for my senile brains.